assets/src/components/Dashboard.tsx

```diff
@@ -421,7 +423,14 @@ const Dashboard = (props: RouteComponentProps) => {
             component={GettingStartedOverview}
           />
           <Route path="/account*" component={AccountOverview} />
-          <Route path="/customers/:id" component={CustomerDetailsPage} />
+          <Route
+            path="/customers/:id"
+            component={
+              isCustomerDetailsV2Enabled
+                ? CustomerDetailsPageV2
+                : CustomerDetailsPage
```

Comment:
As mentioned in the description, I'm hiding the redesign behind an env variable while I work on implementing the rest of the features. I didn't want to have these changes sitting on a branch for too long.

Reply:
perfect! was going to recommend this approach anyway :+1:

Reply:
we could also just create a new temporary route for v2, like `/v2/customers/:id` -- might make it easier to view both 🤷

Reply:
Yeah, that sounds like a good idea! I'll add that. It'll be nice to be able to view it in prod.